http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -143,6 +143,14 @@ public enum RelativeUrlsUsed {
     protected String requestObjectSigningKeyAlias;
     protected String requestObjectSigningKeyStoreType;
     protected JWKEncPublicKeyLocator encryptionPublicKeyLocator;
+    private boolean logoutSessionRequired = true;
+
+    private String postLogoutUri;
+    private boolean sessionRequiredOnLogout = true;
+    private String logoutPath = "/logout";
+    private String logoutCallbackPath = "/logout/callback";
```

Comment:
Would be good to define a constant in Oidc.java for the above two default values.